src/components/Button/Button.vue

```diff
@@ -622,22 +548,18 @@ export default {
 	&--vue-tertiary-no-background {
 		color: var(--color-main-text);
 		background-color: transparent;
-		opacity: .7;
 		&:hover:not(:disabled) {
 			background-color: transparent;
-			opacity: 1;
 		}
 	}
 
 	// Tertiary on primary color (like the header)
 	&--vue-tertiary-on-primary {
 		color: var(--color-primary-text);
 		background-color: transparent;
-		opacity: .7;
```

Comment:
This is then not inline with how default app icons in the topbar look.
Can we skip this from here until the decision about app icons was done?

Reply:
I think it was stated multiple times in in various design calls that we're going to abandon this, cc @jancborchardt & @nimishavijay

Reply:
The app list in the topbar currently does not use this button, so changing this will make it inconsistent, unless the server is adjusted in parallel. So from my POV we should wait for the outcome and then do it in a follow up

Reply:
This shouldn't be related to that special case. See comment there

Reply:
Added special case for tertiary no bg